src/Evolution/Initialization/Initialize.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Parallel/Actions/TerminatePhase.hpp"
+#include "Parallel/AddOptionsToDataBox.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace Initialization {
+/// Available actions to be used in the initialization.
+namespace Actions {
+template <typename OptionsTagsToRemove>
+struct RemoveOptionsFromDataBox {
```

Comment:
There should be exactly zero actions in `Utilities` and `DataStructures`. I'm not sure where they should go, but definitely not in those directories. `NumericalAlgoritms/Actions` is an option, but not super great. Another place in the same situation is `Parallel/Actions` since these actions aren't really related to the parallel code. Again, minimal changes so I'm not really open to moving it.

Reply:
we (tentatively) decided in a call to move these things to `/ParallelAlgorithms/Initialization/` (possibly in another PR)